src/tests/ecalc_cli/snapshots/test_app/test_new_ltp_export_properly/test.INST_A.ltp.tsv

```diff
@@ -1,15 +1,15 @@
-2021	01	33446253	5475000	3889000	70571.61	14235	10539.19	0	306.3675	7.665036	118.6144	8.027098	0.328555	8.633489	38.32487	0	229.9502	30.43615	1.313909	0	131.3999	0	21.89993	1095000	122.3545	0	0	74.46	7.446
-2022	01	693967871	3650000	3518600	1464272	9490	9535.405	0	6356.745	5.110128	107.3173	166.5526	0.218887	7.811172	43.06989	0	229.9501	631.5108	0.87588	0	153.3002	0	21.89998	1095000	130.9912	50.34694	0	79.716	7.9716
-2023	01	533925379	3650000	3518600	1126583	9490	9535.405	0	4890.757	5.110128	107.3173	128.1428	0.218887	7.811172	43.43492	0	214.6201	485.8723	0.87588	0	156.9498	0	20.43985	1022000	130.9912	66.29014	0	79.716	7.9716
-2024	01	311313735	5490000	3806400	656872	14274	10315.34	0	2851.634	7.686037	116.0951	74.71564	0.329455	8.450098	42.82198	0	192.15	283.2951	1.317509	0	151.8899	0	18.30015	914999.9	122.6897	410.5339	32.85216	41.81184	7.4664
-2025	01	411507899	3650001	3591600	868281.6	9490.002	9733.236	0	3769.412	5.109883	109.5438	98.76234	0.219	7.973352	44.895	0	306.6	374.4718	0.876	0	160.5999	0	29.19988	1460000	138.1886	681.5524	37.00224	47.09376	8.4096
-2026	01	292912595	3650001	3591600	618045.6	9490.002	9733.236	0	2683.08	5.109883	109.5438	70.29935	0.219	7.973352	45.26012	0	459.9	266.5507	0.876	0	163.155	0	43.8	2190000	138.1886	468.6232	37.00224	47.09376	8.4096
-2027	01	234469407	3650001	3650001	494730.5	9490.002	9891.505	0	2147.74	5.109883	111.3249	56.27249	0.219	8.103	45.62488	0	268.275	213.3673	0.876	0	163.8849	0	25.55012	1277500	138.1886	354.3396	37.00224	47.09376	8.4096
-2028	01	177817904	5489999	2562000	375195.8	14274	6943.021	0	1628.812	7.686117	78.14112	42.67624	0.329488	5.687611	43.55391	0	153.7201	161.8146	1.317629	0	155.916	0	14.64015	731999.9	125.3475	325.7012	34.01165	43.28755	7.72992
-2029	01	192263657	3650001	2190000	405676.3	9490.002	5934.9	0	1761.135	5.109883	66.795	46.14348	0.219	4.8618	45.26012	0	137.97	174.96	0.876	0	161.6949	0	13.14	657000	134.7266	354.3396	37.00224	47.09376	8.4096
-2030	01	191916726	3650001	2190000	404944.3	9490.002	5934.9	0	1757.958	5.109883	66.795	46.06008	0.219	4.8618	45.62488	0	124.173	174.6446	0.876	0	163.155	0	11.826	591300	133.0609	354.3396	37.00224	47.09376	8.4096
-2031	01	177606484	3650001	2190000	374749.7	9490.002	5934.9	0	1626.876	5.109883	66.795	42.62546	0.219	4.8618	45.62488	0	111.7555	161.622	0.876	0	163.155	0	10.6434	532170	131.5041	347.7193	37.00224	47.09376	8.4096
-2032	01	157104083	5489999	2562000	331489.6	14274	6943.021	0	1439.074	7.686117	78.14112	37.70494	0.329488	5.687611	43.55391	0	100.8403	142.9649	1.317629	0	155.916	0	9.603781	480192	119.4056	316.8373	34.01165	43.28755	7.72992
-2033	01	27668241	0	0	58380	0	0	0	253.441	0	0	6.640445	0	0	0	0	0	25.17807	0	0	0	0	0	0	43.8	0	0	87.6	8.76
+2021-01-01 00:00:00;2022-01-01 00:00:00	2021-01-01 00:00:00;2022-01-01 00:00:00	33446253	5475000	3889000	70571.61	14235	10539.19	0	306.3675	7.665036	118.6144	8.027098	0.328555	8.633489	38.32487	0	229.9502	30.43615	1.313909	0	131.3999	0	21.89993	1095000	122.3545	0	0	74.46	7.446
```

Comment:
Maybe the first two columns needs to be like they were because of export to Centuries?